### DISCUSSION TITLE
Is ipv6 supported?

### DISCUSSION THREAD

**Discussion by trexxeon** [qdm12/gluetun#1550] [Q&A] [answered]
Is ipv6 supported? Mullvad and other vpn providers have support for this but I can't get it to work with wireguard and gluetun, is there any guide for this?

**Comment by axzxc1236** [comment, marked as answer]
I figured out how to use IPv6 + Gluetun + AirVPN: https://github.com/qdm12/gluetun/issues/1648#issuecomment-1607331844
